src/AutoRest.CSharp.V3/CodeGen/CodeWriter.cs

```diff
@@ -202,6 +206,10 @@ public CodeWriter Append(string str)
 
         public CodeWriter Comma() => Append(", ");
 
+        public CodeWriter Space() => Append(" ");
+
+        public CodeWriter Semicolon() => Append(";").Line();
```

Comment:
Maybe `SemicolonLine` would be a better name.

Reply:
My idea was that there is always a line after semicolon

Reply:
Not if a semicolon is used in a for loop. 🤸‍♂